flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayer.java

```diff
@@ -353,6 +355,7 @@ void setStreamUrl(String url, boolean isAssetUrl, boolean autoPlay, long hwAcc)
                 media.addOption(":no-mediacodec-dr");
                 media.addOption(":no-omxil-dr");
             }
+            options.forEach(option -> media.addOption(option));
```

Comment:
see: https://github.com/solid-software/flutter_vlc_player/issues/288#issuecomment-1046482032

this change leads vlc-player doesn't work on Android 6
fix is change this line to orthodox for:

```
if (options != null) {
    for (String option: options) // works everywhere
        media.addOption(option);
}
```

Reply:
Feel free to create PR - I would be happy to merge it!

Reply:
https://github.com/solid-software/flutter_vlc_player/pull/301

Not sure properly decorated...